acceptance/README.md

```diff
@@ -99,6 +99,8 @@ will default to `root`.
 tests as the user for connecting to all hosts using the `winrm` role.
 This user must already be present on the system.
 
+**RUBY_URL** (Default `https://artifactory.delivery.puppetlabs.net/artifactory/generic__buildsources/buildsources/ruby-2.4.4.tar.gz`): The url to download and install ruby from.
```

Comment:
Can you include the ruby.org source as a suggestion? I think it was `https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.4.tar.gz`.

Reply:
Yeah, should I include that this is a mirror of that?

Reply:
I think that's probably clear. Maybe say something like
> If the default mirror is not accessible to you, use `https://cache.ruby-lang.org/pub/ruby/2.4/ruby-2.4.4.tar.gz`